spec/features/support/viewing_responsible_body_info_spec.rb

```diff
@@ -1,32 +1,96 @@
 require 'rails_helper'
 
 RSpec.feature 'Viewing responsible body information in the support area', type: :feature do
-  let(:local_authority) { create(:local_authority, name: 'Coventry') }
+  let(:local_authority) { create(:local_authority, :devolves_management, name: 'Coventry') }
+  let(:local_authority_managing_centrally) { create(:trust, :manages_centrally, :vcap_feature_flag, name: 'Manchester') }
+
   let(:responsible_bodies_page) { PageObjects::Support::ResponsibleBodiesPage.new }
   let(:responsible_body_page) { PageObjects::Support::ResponsibleBodyPage.new }
 
-  scenario 'DfE users see the on-boarded responsible bodies and stats about them' do
-    given_a_responsible_body_with_users
-    and_it_has_some_schools
+  before do
+    stub_computacenter_outgoing_api_calls
+  end
+
+  context 'when no schools are centrally managed' do
+    scenario 'DfE users see the on-boarded responsible bodies and stats about them' do
+      given_a_responsible_body_with_users
+      and_it_has_some_schools
+
+      when_i_sign_in_as_a_dfe_user
+      and_i_visit_the_support_responsible_bodies_page
+      and_i_visit_a_support_responsible_body_page
 
-    when_i_sign_in_as_a_dfe_user
-    and_i_visit_the_support_responsible_bodies_page
-    and_i_visit_a_support_responsible_body_page
+      then_i_can_see_the_users_assigned_to_that_responsible_body
+      and_i_can_see_the_schools_managed_by_that_responsible_body
+    end
 
-    then_i_can_see_the_users_assigned_to_that_responsible_body
-    and_i_can_see_the_schools_managed_by_that_responsible_body
+    scenario 'Computacenter users see the on-boarded responsible bodies and stats about them' do
+      given_a_responsible_body_with_users
+      and_it_has_some_schools
+
+      when_i_sign_in_as_a_computacenter_user
+      and_i_visit_the_support_responsible_bodies_page
+      and_i_visit_a_support_responsible_body_page
+
+      then_i_only_see_the_users_assigned_to_that_responsible_body_who_have_seen_the_privacy_notice
+      and_i_can_see_the_schools_managed_by_that_responsible_body
+    end
   end
 
-  scenario 'Computacenter users see the on-boarded responsible bodies and stats about them' do
-    given_a_responsible_body_with_users
-    and_it_has_some_schools
+  context 'with virtual caps enabled', with_feature_flags: { virtual_caps: 'active' } do
+    context 'when some schools are centrally managed' do
+      scenario 'DfE users see the centrally managed schools' do
+        given_a_centrally_managed_responsible_body_with_users
+        and_it_has_some_centrally_managed_schools
+
+        when_i_sign_in_as_a_dfe_user
+        and_i_visit_the_support_responsible_bodies_page
+        and_i_visit_a_support_responsible_body_that_is_centrally_managed_page
+
+        and_i_can_see_the_schools_that_are_partially_centrally_managed_by_that_responsible_body
```

Comment:
I find the term 'partially_centrally_managed' a bit unclear - it kind of implies that there's different levels of centrally managed for a school. Does it just refer to the fact that some but not all schools are centrally managed, and is there a better way of phrasing this?

Reply:
@aldavidson Yep, I agree, it's not very clear. Thanks, I'll update!

Reply:
I settled on `and_i_can_see_a_mix_of_centrally_managed_and_devolved_schools_by_that_responsible_body`